### DISCUSSION TITLE
Touchpad Disable

### DISCUSSION THREAD

**Discussion by tuneduptechie** [hyprwm/Hyprland#6900] [General]
I have an ASUS TUF A16, running hyprland on arch.
I am currently trying to figure out how to disable my touchpad, beit through keybinding, or when a mouse is connected/disconnected.  I have tried to use the archwiki, to no avail, and a couple different scripts I found on r/Hyprland, and here. Anyone have up to date info? Or how to do this? The touchpad toggler from the AUR doesnt work either as it cant find a synaptics driver even though i installed it.

**Comment by fufexan** [comment]
Resolved in https://github.com/hyprwm/Hyprland/issues/6974.
